.storybook/preview.js

```diff
@@ -5,7 +5,7 @@ import '../packages/all/dist/style/af-toolkit-core.css';
 import '../packages/core/dist/assets/fonts/icons/af-icons.css';
 import './storybook.css';
 
-export const decorators = [addReadme];
+// export const decorators = [addReadme];
```

Comment:
Why comment an export and not delete it if it's no longer used ?

Reply:
fixed